acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/privacy/account/PrivacyAccountSupplier.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.dsl.privacy.account;
+
+import java.util.function.Supplier;
+
+/** Supplier of known funded accounts defined in dev.json */
+public class PrivacyAccountSupplier implements Supplier<PrivacyAccount> {
+
+  public static final PrivacyAccount ALICE = PrivacyAccount.create("key", "orion_key_0.pub");
+  public static final PrivacyAccount BOB = PrivacyAccount.create("key1", "orion_key_1.pub");
+  public static final PrivacyAccount CHARLIE = PrivacyAccount.create("key2", "orion_key_2.pub");
+
+  public static int counter = -1;
```

Comment:
having a counter initialised to -1 is a smell - there should be a better way of using this supplier...